src/common/dsp/AliasOscillator.cpp

```diff
@@ -160,8 +162,14 @@ void AliasOscillator::process_block(float pitch, float drift, bool stereo, bool
 
             phase[u] += phase_increments[u] + fmPhaseShift;
         }
-        output[i] = vL;
-        outputR[i] = vR;
+
+        // bitcrush output
+        auto bits = limit_range(localcopy[oscdata->p[ao_depth].param_id_in_scene].f, 1.f, 8.f);
+        auto quant = powf(2, bits);
```

Comment:
Oh goodness call this pow and divide outside the loop since it is constant across the loop!!

Reply:
Done! That was stupid.